index.html

```diff
@@ -457,6 +457,527 @@ <h1>Authentication and Access Control</h1>
+            input-constrained devices. Similarly, to its <i>parent</i> flow, it requires a close interaction between the
+            resource owner and the WoT consumer. Therefore, the use cases for this flow are the same as the code
+            authorization grant but restricted to all devices that do not have a rich means to interact with the
+            resource owner. However, differently from `code`, RFC 8628 states explicitly that one of the actors of
+            the protocol is an <b>end-user</b> interacting with a <b>browser</b> (even if <a
+              href="https://tools.ietf.org/html/rfc8628#section-6.2">section-6.2</a>
+            briefly describes an authentication using a companion app and BLE), as shown in the following (slightly
+            adapted) diagram:
+
+            <pre>
++----------+
+|          |
+|  Remote  |
+|  Device  |
+|          |
++----^-----+
+     |
+     | (G) Access WoT Affordance
+     |
++----+-----+                                +----------------+
+|          +&gt;---(A)-- Client Identifier ---v+                |
+|          |                                |                |
+|          +&lt;---(B)-- Device Code,      ---&lt;+                |
+|          |          User Code,            |                |
+|   WoT    |          & Verification URI    |                |
+| Consumer |                                |                |
+|          |  [polling]                     |                |
+|          +&gt;---(E)-- Device Code       ---&gt;+                |
+|          |          & Client Identifier   |                |
+|          |                                |  Authorization |
+|          +&lt;---(F)-- Access Token      ---&lt;+     Server     |
++-----+----+   (& Optional Refresh Token)   |                |
+      v                                     |                |
+      :                                     |                |
+     (C) User Code & Verification URI       |                |
+      :                                     |                |
+      ^                                     |                |
++-----+----+                                |                |
+| End User |                                |                |
+|    at    +&lt;---(D)-- End user reviews  ---&gt;+                |
+|  Browser |          authorization request |                |
++----------+                                +----------------+
+</pre>
+            Notable mentions:
+            <ul>
+              <li>the protocol is heavily end-user oriented. In fact, the RFC states the following</li>
+              <ul>
+                <li>Due to the polling nature of this protocol (as specified in Section 3.4), care is needed to avoid
+                  overloading the capacity of the token endpoint. To avoid unneeded requests on the token endpoint, the
+                  client SHOULD only commence a device authorization request when <b>prompted by the user and not

```

Comment:
this section uses SHOULD, MUST in various places, but the document is not normative, so these don't mean the same things as RFC2119.  Somewhere at the top of the document we should mention what these "pseudo-assertions" that are "stating what we feel the best practice should be".   We should create a separate issue for adding that section...